crates/node/builder/src/builder/mod.rs

```diff
@@ -177,6 +177,12 @@ impl<DB, ChainSpec> NodeBuilder<DB, ChainSpec> {
     pub const fn config(&self) -> &NodeConfig<ChainSpec> {
         &self.config
     }
+
+    /// Returns a mutable reference to the node builder's config.
+    #[cfg(feature = "test-utils")]
```

Comment:
we can actually make this non feature gated

Reply:
yeah i updated